vendor/brave-ios/BATBraveRewards.h

```diff
@@ -46,6 +46,20 @@ NS_SWIFT_NAME(BraveRewardsConfiguration)
 
 @end
 
+/// Build configuration around brave rewards for ads & ledger
+NS_SWIFT_NAME(BraveRewardsBuildConfiguration)
+@interface BATBraveRewardsBuildConfiguration : NSObject <NSCopying>
+
+/// The rewards build channel
+@property (nonatomic) BATBraveAdsBuildChannel build_channel;
+
+@property (nonatomic, class, readonly) BATBraveRewardsBuildConfiguration *defaultConfiguration NS_SWIFT_NAME(default);
+@property (nonatomic, class, readonly) BATBraveRewardsBuildConfiguration *releaseChannelConfiguration NS_SWIFT_NAME(releaseChannel);
+@property (nonatomic, class, readonly) BATBraveRewardsBuildConfiguration *betaChannelConfiguration NS_SWIFT_NAME(betaChannel);
+@property (nonatomic, class, readonly) BATBraveRewardsBuildConfiguration *developerChannelConfiguration NS_SWIFT_NAME(developerChannel);
+
+@end
+
```

Comment:
Is there a reason why build channel can't just be added to the `BATBraveRewardsConfiguration` instead of being its own thing?

Reply:
see
```
        let buildConfiguration: BraveRewardsBuildConfiguration
        switch AppConstants.buildChannel {
        case .release, .enterprise:
            buildConfiguration = .releaseChannel
        case .beta:
            buildConfiguration = .betaChannel
        case .dev, .debug:
            buildConfiguration = .developerChannel
        }
```
in `brave-ios/brave-core-1.14.x-ios-move-confirmations-to-ads`

Reply:
Fixed